smpe/bld/smpe.sh

```diff
@@ -169,7 +170,8 @@ do case "$opt" in
   P)   export reqPTF="-P";;
   p)   export VERSION="-p $OPTARG";;
   r)   export ROOT="$OPTARG";;
-  s)   export stopAt="$OPTARG";;
+  S)   export psi="$OPTARG";;
```

Comment:
use the following when not using a second parm for -S
`S)  export psi="-S";;`

Reply:
updated